governance/votes/VoteWizard Exercise Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,195 @@
+| QGov | [0xB0524D8707F76c681901b782372EbeD2d4bA28a6](https://etherscan.io/address/0xB0524D8707F76c681901b782372EbeD2d4bA28a6) | 29.76 |
+| TRUE NAME | [0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d) | 29.76 |
+| UPMaker | [0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd) | 29.76 |
+| vigilant | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) | 29.76 |
+| WBC | [0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47) | 20.16 |
+| PBG | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) | 9.92 |
+| Bandhar | [0xE83B6a503A94a5b764CCF00667689B3a522ABc21](https://etherscan.io/address/0xE83B6a503A94a5b764CCF00667689B3a522ABc21) | 7.68 |
+| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 7.04 |
+|PALC | [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A) | 2.24|
+|Harmony |[0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) |1.92|
+|VoteWizard |[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) |1.6|
+|Navigator |[0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) |0.32|
+
+This is a total of **229.44** MKR.
+
+### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+IF the executive proposal passes, THEN, in accordance with the details in this [post](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6), the [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) (line) will be updated to **0** (zero), signaling the initiation of the `CRVV1ETHSTETH-A` offboarding procedure.
+
+- The Debt Ceiling for `CRVV1ETHSTETH-A` is to be set to **0**.
+- `CRVV1ETHSTETH-A` is to be removed from the automatic line adjustment.
+
+### Funding for Ecosystem Actors
+
+IF the executive proposal passes, THEN, in accordance with the details in this [post](https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306) and the [Support Scope](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a), the following budget streams and one-time DAI transfer will be implemented:
+
+#### Ecosystem Actor Dai Budget Stream
+
+| Entity | Duration | Amount | Wallet Address |
+|--------|----------|--------|----------------|
+| Chronicle Labs Auditor Wallet | 2023-07-01 00:00:00 to 2024-06-30 23:59:59 | 3,721,800 DAI | [0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f) |
```

Comment:
Including times isn't necessary imo. I also tend to prefer actually listing the duration, as I think it's a much more natural way for readers to take in the information.